R/epinow.R

```diff
@@ -78,7 +78,9 @@
 #'
 #' options(old_opts)
 #' }
-epinow <- function(reported_cases,
+# nolint start: cyclocomp_linter
+epinow <- function(data,
+                   reported_cases,
```

Comment:
This will break positional arguments - shall we put `reported_cases` last instead? That would still break a mix of positional and named arguments but perhaps that's rarer?

Reply:
Agreed.